components/earnings/wikis/worldofwarcraft/earnings.lua

```diff
@@ -0,0 +1,16 @@
+---
+-- @Liquipedia
+-- wiki=worldofwarcraft
+-- page=Module:Earnings
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local CustomEarnings = Lua.import('Module:Earnings/Base', {requireDevIfEnabled = true})
+
+CustomEarnings.defaultNumberOfPlayersInTeam = 3
```

Comment:
needed?

WC is on standardized PPT and commons TC so earnings per player should be calculated via `individualprizemoney` (imo)

Reply:
This is WoW, not WC :D

Reply:
This is WoW, not WC :D

Reply:
ah damn